apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -223,6 +232,37 @@ class AppStore extends EventEmitter {
             return false;
         }
         this.unsavedChanges = false;
+
+        if (window.localStorage.getItem('unsavedAction') != null) {
+            if (confirm('You have unsaved changes. Would you like to load them?')) {
```

Comment:
Will discuss more in the main review comment, but I'm feeling more and more that this feature should be a separate PR and be fleshed out a bit more.